plugins/woocommerce-beta-tester/api/features/features.php

```diff
@@ -42,11 +42,7 @@ function toggle_feature( $request ) {
 		return new WP_REST_Response( $features, 204 );
 	}
 
-	if ( isset( $custom_feature_values[ $feature_name ] ) ) {
-		unset( $custom_feature_values[ $feature_name ] );
-	} else {
-		$custom_feature_values[ $feature_name ] = ! $features[ $feature_name ];
-	}
+	$custom_feature_values[ $feature_name ] = ! $features[ $feature_name ];
```

Comment:
I removed the `if` condition and directly set the value to the opposite of the current value. In this way, we can persist the value in the database and use it even when the feature config has yet to be loaded.